### DISCUSSION TITLE
Some tab issues

### DISCUSSION THREAD

**Discussion by anttiuauau** [Godiesc/firefox-gx#206] [Q&A]
Hey guys, I'm having some problems with the tabs.I don't know how to explain what's happening,. I hope someone could help me. I mean, it is not a big deal but it is kind of a bummer. 
![aaaaaaaaaaa](https://github.com/user-attachments/assets/4fe74c1f-a689-4cc4-ac56-8be5fdbdd15f)

P.D.: I love this theme, i really love it, so thank you guys for making this.

**Comment by anttiuauau** [comment]
So, as you see, the tabs are surrounded by red line that keeps them from merging with the rest of the browser while open (I think that can be a way to explain it)

**Reply by Godiesc** [reply]
Hi, I answered in the issue comments: https://github.com/Godiesc/firefox-gx/issues/207#issuecomment-2411624338
